src/styles/app.css

```diff
@@ -88,7 +88,7 @@ a.qpp-docs-title-back {
 
 .usa-sidenav-list li {
   border-top: none;
-  font-size: 1rem;
+  font-size: inherit;
```

Comment:
if we exclude the font-size rule altogether, will it inherit font-size by default?